ngraph/test/backend/reshape.in.cpp

```diff
@@ -234,217 +243,13 @@ NGRAPH_TEST(${BACKEND_NAME}, reshape_m2m_same)
                                   MIN_FLOAT_TOLERANCE_BITS));
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, reshape_m2m_transpose)
```

Comment:
Why didn't you migrate these tests on v1 reshape?

Reply:
because v1::Reshape (in contrast to v0::Reshape) does not perform transpose.